meshnesh/android-level-up
# #156185242 Unit /UI Test

# What does this PR do?

* Implement Unit Tests
* Implement UI Test

## Description of Task to be completed?

Create Both Unit and UI tests for the application as a proof of concept it works.

## How should this be manually tested?

* Run the tests the user can run the command, ensure you are connected to running device for it to work
 `./gradlew createDebugCoverageReport` 

* After which the user can open the access the following directory `app/build/reports/androidTests` for individual tests and `app/build/reports/coverage` for the total test coverage.

## What are the relevant pivotal tracker stories?

[Unit Test #156185242](https://www.pivotaltracker.com/story/show/156185242)
[UI Test #157354404](https://www.pivotaltracker.com/story/show/157354404)

Screenshots (if appropriate)
N/A
